org.envirocar.app/res/layout/activity_car_selection_layout_carlist_entry.xml

```diff
@@ -13,41 +13,39 @@
     You should have received a copy of the GNU General Public License along
     with the enviroCar app. If not, see http://www.gnu.org/licenses/.
 -->
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="match_parent"
-    android:layout_height="64dp"
+<com.google.android.material.card.MaterialCardView
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:layout_marginTop="@dimen/spacing_large"
-    android:layout_marginLeft="@dimen/spacing_large"
-    android:layout_marginRight="@dimen/spacing_large"
-    android:orientation="vertical">
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    app:cardCornerRadius="@dimen/cardview_corner_radius"
```

Comment:
I think the definition of this dimension is missing within the [dimens.xml](https://github.com/enviroCar/enviroCar-app/blob/master/org.envirocar.app/res/values/dimens.xml). Thus, building fails. Can you fix it and add the missing dimension?

Reply:
Yes Sure.